Lib/test/test_capi/test_opt.py

```diff
@@ -1925,6 +1925,34 @@ def testfunc(n):
         self.assertNotIn("_GUARD_NOS_INT", uops)
         self.assertNotIn("_GUARD_TOS_INT", uops)
 
+    def test_get_len(self):
```

Comment:
You are missing the test case that Brandt suggested: https://github.com/python/cpython/pull/133345#discussion_r2080344111

You could also split `test_get_len` into two separate test cases - other test cases typically only have one `testfunc` :)

Reply:
I think we need two case here

1. `match tuple`, the `GET_LEN` will return a const
2. `match non-tuple`, the `GET_LEN` will return a non-const lang 

I have split this into two test case. Plz correct me if I'm wrong. PTAL when you get time

Reply:
If I understood correctly, Brandt was suggesting a test case with a non-constant tuple. That is, a tuple with non-constant values, which is different from a non-tuple. So for example `(object(), object())` (non-constant tuple) rather than `[1, 2]` (non-tuple) :slightly_smiling_face:

Reply:
> If I understood correctly, Brandt was suggesting a test case with a non-constant tuple.

Yes, this is the original version of my thought. But I find that the code we add the the `_GET_LEN` will effect between `tuple` and `non tuple`. So I modify the test

But, your idea is making sense. Let me add test for non-constant tuple

Reply:
More tests doesn't hurt! :)